src/traces/indicator/index.js

```diff
@@ -24,7 +24,11 @@ module.exports = {
 
     meta: {
         description: [
-            'TODO: add description'
+            'An indicator is used to visualize single a `value` along with some',
```

Comment:
`to visualize single a` :arrow_right: `to visualize a single`

Reply:
fixed!